mitosheet/src/utils/formatColumns.tsx

```diff
@@ -1,346 +0,0 @@
-            switch(columnFormatType.type) {
-                case FormatType.DEFAULT:
-                    if (columnDtype?.includes('int')) {
-                        // If the column is an int, default to 0 decimal places
-                        return formatCellDataAsStringWithCommas(cellData, 0)
-                    } else {
-                        // We show the full number of decimals if it is a float
-                        const cellDataToParse = '' + cellData;
-                        let numDecimals = 0;
-                        if (cellDataToParse.includes('.')) {
-                            numDecimals = cellDataToParse.toString().split(".")[1].length || 0;
-                        }
-                        return formatCellDataAsStringWithCommas(cellData, numDecimals)
-                    }
-                case FormatType.PLAIN_TEXT:
-                    return '' + cellData
-                case FormatType.ROUND_DECIMALS: {
-                    const numDecimals = columnFormatType.numDecimals
-                    return Number(cellData).toLocaleString("en-US", {minimumFractionDigits: numDecimals, maximumFractionDigits: numDecimals})
-                }
-                case FormatType.PERCENTAGE: {
-                    return Number(cellData).toLocaleString("en-US", {style: 'percent', minimumFractionDigits: 2, maximumFractionDigits: 2})
-                }
-                case FormatType.ACCOUNTING: {
-                    return Number(cellData).toLocaleString("en-US", {style: "currency", currency: "USD", currencySign: "accounting"})
-                }
-                case FormatType.K_M_B: {
-                    // Ensure we're operating on a number.
-                    const number = Number.parseFloat(String(cellData))
-                    const number_abs = Math.abs(number)
-                    if (number_abs >= 1000000000) {
-                        return (number / 1000000000).toFixed(1).replace(/\.0$/, '') + 'B';
-                    } else if (number_abs >= 1000000) {
-                        return (number / 1000000).toFixed(1).replace(/\.0$/, '') + 'M';
-                    } else if (number_abs >= 995) {
-                        return (number / 1000).toFixed(1).replace(/\.0$/, '') + 'K';
-                    } else {
-                        // We make anything less than ABS(995) displayed as 0k to match Excel formatting 
-                        return '0k';
-                    }
-                }
-                case FormatType.SCIENTIFIC_NOTATION:
-                    return Number.parseFloat(String(cellData)).toExponential(2);
-            }
-        } else {
-            return '' + cellData
-        }
-    } catch {
-        return '' + cellData 
-    }
-    
-}
-
-/* 
-    Sends the changeColumnFormat message to the backend so that the format of the selected columns
-    will be updated in the state.
-*/
-export const changeFormatOfSelectedColumns = async (
```

Comment:
This function is extremely thin. It makes one function call, and then calls another really similar function. 

This probably should signify this doesn't make sense as a function at all - just do the getSelectedNumberSeriesColumnIDs in the call location, and remove 30 lines of code!